### DISCUSSION TITLE
v0.0.0-alpha.5

### DISCUSSION THREAD

**Discussion by Losses** [Losses/rune#81] [Announcements]
# Features
* Configure CI for Steam Sniper container by @abc1763613206 in https://github.com/Losses/rune/pull/76
* Speed up analysis functionality  by @XMLHexagram in https://github.com/Losses/rune/pull/77
* Implement refresh list feature
* Add "Create Mix" and "Create Playlist" options to the context menu
* Add a confirmation modal for the delete library list feature
* Implement multi-language selection
* Add official support for Japanese and Traditional Chinese by @NovaDNG in https://github.com/Losses/rune/pull/78
* Add experimental support for German, French, Italian, Korean, Ukrainian, Hokkien, and Cantonese

## Bugs
* Fix Android compilation errors
* Insert checks in the "Favorites" track API to prevent write errors
* Play Next option in Add to Queue setting not working by @XMLHexagram in https://github.com/Losses/rune/pull/80

**Full Changelog**: https://github.com/Losses/rune/compare/v0.0.0-alpha.4...v0.0.0-alpha.5

<hr /><em>This discussion was created from the release <a href='https://github.com/Losses/rune/releases/tag/v0.0.0-alpha.5'>v0.0.0-alpha.5</a>.</em>

**Comment by hydroid7** [comment]
Hi,
I downloaded the compiled binary for my system (Archlinux). If I try to start rune, I get following error in the console and a black screen:
```
(rune:200116): Gtk-WARNING **: 11:30:05.619: Theme parsing error: main-light.css:7893:33: '/*' in comment block

(rune:200116): Gtk-WARNING **: 11:30:05.619: Theme parsing error: main-light.css:7893:55: Expected a valid selector
2024-11-18T10:30:06.162365Z  INFO flutter: Device is not Windows 10, skip the patch
2024-11-18T10:30:06.163547Z  INFO flutter: Initializing config file at: /home/user/.local/share/ci.not.rune
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPlatformDirectoryException(Unable to get application documents directory)
#0      GetStorage._init (package:get_storage/src/storage_impl.dart:49)
<asynchronous suspension>
#1      new GetStorage._internal.<anonymous closure> (package:get_storage/src/storage_impl.dart:30)
<asynchronous suspension>
#2      SettingsManager._init (package:rune/utils/settings_manager.dart:39)
<asynchronous suspension>
#3      SettingsManager.getValue (package:rune/utils/settings_manager.dart:45)
<asynchronous suspension>
#4      main (package:rune/main.dart:83)
<asynchronous suspension>

```
Can you please give me any hint how to start the application? Thank you in advance!

**Reply by Losses** [reply]
Oops! I guess `xdg-user-dir` is missing in your system! Can you confirm this?

**Reply by hydroid7** [reply]
You're right! :+1: 
Thank you for the hint. I solved it by installing the `xdg-user-dirs` package.
